src/components/common/Actions.vue

```diff
@@ -90,13 +93,15 @@ export default class Actions extends Vue {
   @Action setIsSavingResuming!: ActionBindingIF
   @Action setIsFilingPaying!: ActionBindingIF
 
+  // Local Properties
+  private authUrl = sessionStorage.getItem('AUTH_URL') || ''
+
   /**
    * Method called when Cancel button is clicked.
    */
   private onCancel (): void {
-    const authUrl = sessionStorage.getItem('AUTH_URL') || ''
     // assume Auth URL is always reachable
-    window.location.assign(authUrl)
+    window.location.assign(this.authUrl)
```

Comment:
FYI, I'm not sure about this one. I'd say we redirect to a dashboard, but does it matter if we have saved a draft -- different dashboard??? Dunno atm.
(no change required)

Reply:
Definitely, future state will be elsewhere.